README.md

```diff
@@ -26,7 +26,7 @@ You will need the publish and subscribe keys to authenticate your app. Get your
 
    * for Gradle, add the following dependency in your `gradle.build`:
```

Comment:
if we're changing it to the Kotlinscript format (which is recommended) in that case this should be `gradle.build.kts`